app/AboutAppearanceViewController.m

```diff
@@ -158,4 +177,15 @@ - (IBAction)fontSizeChanged:(UIStepper *)sender {
     UserPreferences.shared.fontSize = @((int) sender.value);
 }
 
+- (void) setStatusBar:(id)sender {
+    UISwitch *switchy = sender;
```

Comment:
typo, or cute diminutive?

Reply:
Cute UwU... I'm not too sure whether iSH does cute yet so I am testing the waters.